app.py

```diff
@@ -30,7 +37,9 @@
 elif tabs == 'Customers':
     customer_page = DataPage("Customers", "CustomerID", st.session_state.customer_dict)
     st.session_state.customer_dict = customer_page.show_page()
+    create_customers_in_db(st.session_state.customer_dict.to_dict(orient='records'))
```

Comment:
Trochę mi nie pasuje żeby tak to wyglądało
Chodzi o to w jaki sposób działa biblioteka `streamlit`
Na ten moment ja bym został przy założeniu że do momentu puszczenia programu, dane zostają trzymane w tabelach jako suche dane a nie w bazie lub klasach.
Aby poprawnie to działało trzeba albo zmienić rodzaj tabeli albo napisać callback do niej